### ISSUE TITLE
Example on how to draw with this

### ISSUE THREAD

**Issue by sandmor** [RazrFalcon/rustybuzz#25]
So, I never use harfbuzz and I'm a bit lost about how to use this and _integrate_ with ttf-parser or rusttype, can you provide an example of using this to draw non-latin text, I'm talking of an example with tiny-skia, ttf-parser and this or something that you want.
Thank you in advance

**Comment by RazrFalcon** [comment]
Yes, I've planned to write such example. It would be a bit rudimentary, because font fallback and emojis must be handled by the caller, but other than that it should be fine.

**Comment by xobs** [comment]
I have a terrible, awful repo up at https://git.xobs.io/xobs/rust-font-test that you're welcome to steal from. That repository is an experiment in memory usage and rendering.

It uses rusttype for rendering.

It even renders emoji properly, and does a decent job of Arabic.

**Comment by sandmor** [comment]
@xobs Thank you

**Comment by rnikander** [comment]
When I call `shape`, should the `GlyphPosition`s I get back have `y_offset` set to something other than zero? Mine are all zero. I'm just trying to draw a simple string with this and ab_glyph. And I can't figure out how to shift the glyphs in the Y dimension.

**Comment by asibahi** [comment]
I used rustybuzz and ab_glyph in a project of mine. Code should be here https://github.com/asibahi/nun/blob/master/src/draw.rs
